.travis.yml

```diff
@@ -53,6 +53,8 @@ addons:
   apt:
     packages:
     - nsis
+    - imagemagick
```

Comment:
I don't think `imagemagick` is used anymore and it could be removed here
https://github.com/streamlink/streamlink/pull/2165/commits/c0b0ac68bfe23f86d34dab1fca3406e5ac228894